cg_hermes/config/taxprofiler.py

```diff
@@ -0,0 +1,72 @@
+"""Tags that are defined in Taxprofiler deliverables mapped to tags stored in Housekeeper."""
+
+from cg_hermes.config.nextflow import NEXTFLOW_TAGS
+
+TAXPROFILER_COMMON_TAGS = {
+    frozenset({"kraken2_combined_report", "kraken2"}): {
+        "is_mandatory": True,
+        "bundle_id": True,
+        "tags": ["kraken2", "metagenomics-report"],
```

Comment:
Is this tag correct? Shouldn't it be `combined-report` given the taxprofiler tag `kraken2_combined_report`?

Reply:
You are right!! Update the PR.